src/hmem_cuda.c

```diff
@@ -538,6 +542,13 @@ static int cuda_hmem_dl_init(void)
 		goto err_dlclose_cuda;
 	}
 
+	cuda_ops.cudaDeviceFlushGPUDirectRDMAWrites = dlsym(cudart_handle,
```

Comment:
do you need to check cuda version here? This will fail for cuda <=11.3 right?

Reply:
Bump comment.

Reply:
I did it elsewhere but I can bring it here... I think thats better here.

Reply:
Done on next push.